NorthstarDedicatedTest/version.cpp

```diff
@@ -4,62 +4,27 @@
 char version[16];
 char NSUserAgent[32];
 
+// Turning off clang-format here so it doesn't mess with style as it needs to be this way for CI
+// clang-format off
+#define NORTHSTAR_VERSION 0,0,0,1
+// clang-format on
+
 void InitialiseVersion()
 {
-	HRSRC hResInfo;
-	DWORD dwSize;
-	HGLOBAL hResData;
-	LPVOID pRes, pResCopy;
-	UINT uLen = 0;
-	VS_FIXEDFILEINFO* lpFfi = NULL;
-	HINSTANCE hInst = ::GetModuleHandle(NULL);
+	int northstar_version[] = {NORTHSTAR_VERSION};
```

Comment:
This could be turned into a `constexpr` which saves a bit of time on startup

Reply:
So
```C
constexpr int northstar_version[] = {NORTHSTAR_VERSION};
```
?

Reply:
yes, but you would put that in global scope, outside of the function. I'm not exactly sure on the semantics of `constexpr` mixed with `#define`, but i *think* that should be fine. The easiest way to check is to simply inspect the value of `northstar_version` in the binary, since it would be placed somewhere in the `.data` section

Reply:
Okay so it turns out the semantics for `constexpr` are slightly different to normal construction. This should work however:
`constexpr int ns_version[4]{NORTHSTAR_VERSION};`

Reply:
The above can be verified by running a shimmed version through gdb and running `print (int[4])ns_version` before ever starting execution.

Reply:
Done in f3e897146521db5db3900bef9280299b48a5af6a